### ISSUE TITLE
Frage: Lokale Ressourcen?

### ISSUE THREAD

**Issue by mploigt** [empeiria/keeptalkinggame-german#9]
Wäre es nicht besser, wenn alle Ressourcen lokal vorliegen würden? Momentan wird ja alles von http://www.bombmanual.com gezogen. Aber was passiert, wenn die Seite irgendwann mal nicht mehr da ist?

**Comment by mploigt** [comment]
Ich habe das in meinem Fork schon mal so gemacht, klappt gut: https://mploigt.github.io/keeptalkinggame-german

**Comment by joerglohrer** [comment]
Kann man so machen - ist halt eine Frage der Lizenzierung / des Urheberrechts. 
Wenn die Keep-talking-Leute das unter Creative Commons stellen würden, wäre es unproblematisch

**Comment by mploigt** [comment]
Stimmt, in der offiziellen Version sollte man das dann lieber so lassen :)
